src/views/SignUp.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <v-row align="center" justify="center">
+    <v-col cols="12" sm="8" md="4">
+      <v-card class="elevation-12">
+        <v-toolbar
+          :light="$vuetify.theme.dark"
+          :dark="!$vuetify.theme.dark"
+          flat
+        >
+          <v-toolbar-title>Sign Up</v-toolbar-title>
+        </v-toolbar>
+        <v-card-text>
+          <v-form lazy-validation v-model="valid" ref="signUpForm">
```

Comment:
Would be so cool if this works:

```suggestion
          <v-form lazy-validation v-model="valid" ref="signUpForm" v-on:keyup.enter="handleSubmit">
```